backend/game/dice/dice_handler_test.py

```diff
@@ -18,7 +18,6 @@
 MOCK_DIE_ROLL_RESULT_B = dice.DieValue.ENERGY
 
 dice.roll = Mock(return_value=MOCK_DIE_ROLL_RESULT)
-dice.roll_many = Mock(return_value=MOCK_SIX_DIE_VALUES_A)
```

Comment:
This mock breaks tests in Giant brain since any morll_many returned 6 dice even through user was entitled to 7